sormas-ui/src/main/java/de/symeda/sormas/ui/configuration/DevModeView.java

```diff
@@ -508,6 +647,11 @@ private void generateContacts() {
 		ContactGenerationConfig config = contactGeneratorConfigBinder.getBean();
 
 		List<Disease> diseases = config.getDisease() == null ? FacadeProvider.getDiseaseConfigurationFacade().getAllDiseases(true, true, true) : null;
+		Disease disease = config.getDisease();
```

Comment:
Move this definition one line up, you can then use it as well in the definition of `List<Disease> diseases`